src/models/authentication.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, Integer, String, Boolean
+
+from data.database import Base
+
+
+class User(Base):
+    __tablename__ = "users"
+
+    id = Column(Integer, primary_key=True, index=True)
+    tenant_id = Column(Integer, index=True) # Not sure about this one
```

Comment:
I don't see DH being multi-tenant anytime soon. I would see RBAC fit the bill for most of the things, so I would add Roles"

Having said that, leave it. JIC.

Reply:
Yes, I will make it nullable and optional so it does not slowdown the implementation